### ISSUE TITLE
Internationalization

### ISSUE THREAD

**Issue by amannn** [shadcn/taxonomy#56]
Would you be interested in supporting i18n in this app? I'm the maintainer of [`next-intl`](https://next-intl-docs.vercel.app/), an internationalization library for Next.js. I just added support for Next.js 13 and the `app` folder in the latest version ([docs](https://next-intl-docs.vercel.app/docs/next-13)).

The latest version includes a drop-in replacement for the i18n routing that Next.js previously provided, but which was removed for the `app` folder. At this time, Client Components are fully supported by `next-intl` and support for Server Components is in preview for now.

If you'd be interested, I could help to set this up in Taxonomy—just let me know!

**Comment by shadcn** [comment]
Sure. Let me check it out.

**Comment by Just-Moh-it** [comment]
@shadcn let's have this as a priority, please 🙏🙂

**Comment by corysimmons** [comment]
Does anyone know if there is a nice/fair comparison table for all the i18n libs out there?

**Comment by GllgaMesh** [comment]
I made it work with i18next

If someone interested 

[](https://github.com/i18next/next-13-app-dir-i18next-example-ts)

**Comment by benyamynbrkyc** [comment]
Any updates on [`next-intl`](https://next-intl-docs.vercel.app/)?

**Comment by GllgaMesh** [comment]
> Any updates on [`next-intl`](https://next-intl-docs.vercel.app/)?
better use nextjs built-in internalization options, i18next have issues with client rendering, found this example below, all works pretty good
https://github.com/romandesigns/nextjs-13-1-app-dir-i18n-example/tree/main

**Comment by benyamynbrkyc** [comment]
> > Any updates on [`next-intl`](https://next-intl-docs.vercel.app/)?<
> 
> better use nextjs built-in internalization options, i18next have issues with client rendering, found this example below, all works pretty good [example](https://github.com/romandesigns/nextjs-13-1-app-dir-i18n-example/tree/main)

Just checked it out, it is pretty good. I see the translations are sourced from JSON files in the dictionaries folder. Is the preferred way of managing these files something like a build step? So far, I've just written scripts to fetch/update the files. Is there another, perhaps more principled way?

**Comment by GllgaMesh** [comment]
> > > Any updates on [`next-intl`](https://next-intl-docs.vercel.app/)?<
> > 
> > 
> > better use nextjs built-in internalization options, i18next have issues with client rendering, found this example below, all works pretty good [example](https://github.com/romandesigns/nextjs-13-1-app-dir-i18n-example/tree/main)
> 
> Just checked it out, it is pretty good. I see the translations are sourced from JSON files in the dictionaries folder. Is the preferred way of managing these files something like a build step? So far, I've just written scripts to fetch/update the files. Is there another, perhaps more principled way?

I don't think so, I made it from his example, but most of it was also written on the beta version of nextjs docs, some must be written, and some can depend on your preference and works pretty well for me. I tried some versions, the best option I found for now is this one.

**Comment by BearCooder** [comment]
Now that [next-intl v3](https://next-intl-docs.vercel.app/blog/next-intl-3-0) released in Novemver 2023 with stable Server Components support what is the status of this?
